tests/test_org_sessions_api.py

```diff
@@ -0,0 +1,165 @@
+
+        # values for test data
+        self.minutes = 180
+        self.skew = timedelta(minutes=self.minutes)
+        self.started = datetime.utcnow().replace(
+            minute=0, second=0, microsecond=0, tzinfo=pytz.utc
+        )
+
+        self.storage = get_writable_storage(StorageKey.SESSIONS_RAW)
+
+        self.id_iter = itertools.count()
+        next(self.id_iter)  # skip 0
+        self.project_id = next(self.id_iter)
+        self.org_id = next(self.id_iter)
+        self.project_id2 = next(self.id_iter)
+
+        self.generate_session_events(self.org_id, self.project_id)
+        self.generate_session_events(self.org_id, self.project_id2)
+
+    def generate_session_events(self, org_id, project_id: int) -> None:
+        processor = self.storage.get_table_writer().get_stream_loader().get_processor()
+        meta = KafkaMessageMetadata(
+            offset=1, partition=2, timestamp=datetime(1970, 1, 1)
+        )
+        distinct_id = uuid4().hex
+        template = {
+            "session_id": uuid4().hex,
+            "distinct_id": distinct_id,
+            "duration": None,
+            "environment": "production",
+            "org_id": org_id,
+            "project_id": project_id,
+            "release": "sentry-test@1.0.1",
+            "retention_days": settings.DEFAULT_RETENTION_DAYS,
+            "seq": 0,
+            "errors": 0,
+            "received": datetime.utcnow().timestamp(),
+            "started": self.started.timestamp(),
+        }
+        events = [
+            processor.process_message(
+                {
+                    **template,
+                    "status": "exited",
+                    "duration": 1947.49,
+                    "session_id": uuid4().hex,  # "8333339f-5675-4f89-a9a0-1c935255ab58",
+                    "started": (self.started + timedelta(minutes=13)).timestamp(),
+                },
+                meta,
+            ),
+            processor.process_message(
+                {**template, "status": "exited", "quantity": 5}, meta,
+            ),
+            processor.process_message(
+                {**template, "status": "errored", "errors": 1, "quantity": 2}, meta,
+            ),
+            processor.process_message(
+                {
+                    **template,
+                    "distinct_id": distinct_id,
+                    "status": "errored",
+                    "errors": 1,
+                    "quantity": 2,
+                    "started": (self.started + timedelta(minutes=24)).timestamp(),
+                },
+                meta,
+            ),
+        ]
+        filtered = [e for e in events if e]
+        write_processed_messages(self.storage, filtered)
+
+    def test_simple(self) -> None:
+        query = Query(
+            dataset="sessions",
+            match=Entity("org_sessions"),
+            select=[
+                Column("org_id"),
+                Function("groupUniqArray", [Column("project_id")], "project_ids"),
```

Comment:
Are you going to use `groupUniqArray` in your code? Or just use group by like we talked about?

Reply:
I'm going to try using group by like we talked about (and will change this test to reflect that)